docs/abbreviations.md

```diff
@@ -25,15 +25,15 @@
 *[IP]: Internet protocol address
 *[IPs]: Internet protocol addresses
 *[CPU]: Central processing unit, also known as "processor"
-*[CLI]: Command Line Interface
+*[CLI]: Command-line Interface
 *[OS]: Operating system
 *[GCC]: the GNU compiler collection (compiler system produced by the GNU Project supporting various programming languages)
 *[repo]: Repository on GitHub
 *[RegEx]: Regular expression
 *[Regex]: Regular expression
 *[regex]: Regular expression
-*[SQLite3]: Database engine that handles an SQL databases in a file
```

Comment:
Also, not sure about this.

Reply:
Not sure that it's an abbreviation? Or not sure if `an SQL database` is right? If the latter then I use `an SQL` since the `S` is an `ess` sound.

Reply:
My concern here is about the "in a file" part. The other looks good to me.

Reply:
SQLite3 uses flat files, that's one of the primary points for its use. Doesn't require a daemon.